ui/packages/consul-peerings/app/components/consul/peer/form/index.hbs

```diff
@@ -1,63 +1,50 @@
-<div
-  class={{class-map
-    'consul-peer-form'
-  }}
-  ...attributes
->
+<div class={{class-map "consul-peer-form"}} ...attributes>
   <StateMachine
-    @src={{require './chart.xstate' from='/components/consul/peer/form'}}
-  as |fsm|>
+    @src={{require "./chart.xstate" from="/components/consul/peer/form"}}
+    as |fsm|
+  >
 
     <TabNav
       @items={{array
         (hash
-          label='Generate token'
-          selected=(state-matches fsm.state 'generate')
+          label="Generate token"
+          selected=(state-matches fsm.state "generate")
           state="GENERATE"
         )
         (hash
-          label='Establish peering'
-          selected=(state-matches fsm.state 'initiate')
+          label="Establish peering"
```

Comment:
Do we want translation keys for these?

Reply:
Yes, that is a good idea. I had just ran the prettier formatter on this, but I think it's worth adding the copy here to translations. Thanks for mentioning it!